regression/mint/react-icu/l10nmonster.mjs

```diff
@@ -5,34 +5,34 @@ export default class ReactConfig {
     unqualifiedPenalty = 9;
 
     constructor({ ctx, stores, adapters, filters, normalizers, translators }) {
-        this.source = new adapters.FsSource({
-            globs: [ '**/en/*.json' ],
-            targetLangs: [ 'de', 'ru' ],
-        });
-        this.resourceFilter = new filters.JsonFilter();
-        this.decoders = [ normalizers.iosPHDecoder, normalizers.javaEscapesDecoder ];
-        // this.translationProvider = new translators.TranslationOS({
-        //     baseURL: 'https://api-sandbox.translated.com/v2',
-        //     apiKey: ctx.env.translated_api_key_sandbox,
-        //     serviceType: 'premium',
-        //     quality: 90,
-        //     trafficStore: new stores.FSTrafficStore(),
-        // });
-        // this.translationProvider = new translators.ModernMT({
-        //     apiKey: ctx.env.mmt_api_key,
-        //     quality: 40,
-        //     maxCharLength: 100,
-        // });
-        this.translationProvider = new translators.DeepL({
-            apiKey: ctx.env.deepl_api_key,
-            quality: 40,
-        });
-        this.target = new adapters.FsTarget({
-            targetPath: (lang, resourceId) => resourceId.replace('en/', `${lang}/`),
+        this.translationProvider = new translators.TranslationOS({
```

Comment:
it's ok to have external providers that require keys in the samples, but I don't think it should be required to run regression. (we should add regression as a github action to confirm that!) 

also, if there's any bug and some translations do end up being pushed, they'll end up to a third party.